### PR TITLE
Pegasus: migration to add displayweight_pre/_25 to tutorials

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

**Comment by breville** [comment]
We will leave _k5 intact until we've migrated schema & code, but I'm adding _pre and _25 in this migration and we can add code to start using them once they've hit production.  Then we can deprecate use of _k5 and then remove it from the schema.  Once all that's done we can remove _k5 from the gsheet.

**Comment by tanyaparker** [comment]
SGTM

File: pegasus/migrations/124_add_more_grade_displayweights_to_tutorials.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :displayweight_pre, Integer, null: false
+      add_column :displayweight_25, Integer, null: false
```
Comment (Erin007) [inline pegasus/migrations/124_add_more_grade_displayweights_to_tutorials.rb:5]: 25 here refers to grades 2-5?

Comment (breville) [reply pegasus/migrations/124_add_more_grade_displayweights_to_tutorials.rb:5]: Yes.

Comment (breville) [reply pegasus/migrations/124_add_more_grade_displayweights_to_tutorials.rb:5]: Similar to how k5 is used for K-5.

**Comment by breville** [comment]
circleci failure is known and unrelated.  merging.

**Pull request merged**